playbooks/provision.yml

```diff
@@ -79,6 +86,12 @@
             capi_cluster_kubeconfig_path: "{{ ansible_env.HOME }}/kubeconfig-{{ capi_cluster_release_name }}.yaml"
       when: install_mode == 'ha'
 
+    # Since capi_cluster role already installs vanilla kube-prometheus-stack, we can include this role AFTER the
+    # capi_cluster role so that HA deployments will simply receive an updated chart with additional chart values
+    # which include monitoring and alerting config and so that singlenode/demo deployments will also have the 
+    # same monitoring stack installed from scratch
+    - include_role:
+        name: stackhpc.azimuth_ops.kube_prometheus_stack
```

Comment:
Doing this on a HA cluster will result in the `HelmRelease` objects being inconsistent with the actual Helm release, which may even result in the changes being undone by the addon operator.

We should either:

  1. Not do the monitoring on single node (I know @JohnGarbutt objects to that solution!)
  2. Only do this for single node deployments
  3. Disable the monitoring from the openstack-cluster Helm chart on the HA cluster and install our own in both single node and HA cases